privacy/gdpr/gdpr.go

```diff
@@ -0,0 +1,34 @@
+package gdpr
+
+import (
+	"encoding/json"
+
+	"github.com/buger/jsonparser"
+	"github.com/mxmCherry/openrtb"
+)
+
+// Policy represents the GDPR regulation of an OpenRTB bid request.
+type Policy struct {
+	Enabled bool
```

Comment:
Is 'Enabled' the right term here? .. or perhaps 'Present' would be better?

Reply:
In order to make it even more obvious that it is a boolean value, why not `isPresent` or `isDefined`?

Reply:
I like `IsPresent`. I've been back and forth on if it's really needed. I'm heaving leaning towards including it so the information is captured and made available for other parts of the code to consume.

Reply:
Eh. I think I'll just use pointers for now. That's my best guess on how this sort of thing is usually done with Go, since these values are a 1:1 mapping to JSON structures.

Reply:
Nope. That didn't work out nicely.